modules/log6x-logging-clo-cli-install.adoc

```diff
@@ -0,0 +1,163 @@
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a service account to be used by the log collector:
++
+[source,terminal]
+----
+$ oc create sa logging-collector -n openshift-logging
+----
+
+. Assign the necessary permissions to the service account for the collector to be able to collect and forward logs. In this example, the collector is provided permissions to collect logs from both infrastructure and application logs.
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user logging-collector-logs-writer -z logging-collector -n openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z logging-collector -n openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z logging-collector -n openshift-logging
+----
+
+. Create a `ClusterLogForwarder` CR:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging # <1>
+spec:
+  serviceAccount:
+    name: logging-collector # <2>
+  outputs:
+  - name: lokistack-out
+    type: lokiStack # <3>
+    lokiStack:
+      target: # <4>
+        name: logging-loki 
+        namespace: openshift-logging
+      authentication:
+        token:
+          from: serviceAccount
+    tls:
+      ca:
+        key: service-ca.crt
+        configMapName: openshift-service-ca.crt
+  pipelines:
+  - name: infra-app-logs
+    inputRefs: # <5>
+    - application
+    - infrastructure
+    outputRefs:
+    - lokistack-out
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify the name of the service account created before.
+<3> Select the `lokiStack` output type to send logs to the `LokiStack` instance.
+<4> Point the `ClusterLogForwarder` to the `LokiStack` instance created earlier.
+<5> Select the log output types you want to send to the `LokiStack` instance.
+
+. Apply the `ClusterLogForwarder CR` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+.Verification
+
+. Verify the installation by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-logging
+----
++
+.Example output
+[source,terminal]
+----
+$ oc get pods -n openshift-logging
+NAME                                               READY   STATUS    RESTARTS   AGE
+cluster-logging-operator-fb7f7cf69-8jsbq           1/1     Running   0          98m
+collector-222js                                    2/2     Running   0          18m
+collector-g9ddv                                    2/2     Running   0          18m
+collector-hfqq8                                    2/2     Running   0          18m
+collector-sphwg                                    2/2     Running   0          18m
+collector-vv7zn                                    2/2     Running   0          18m
+collector-wk5zz                                    2/2     Running   0          18m
```

Comment:
Since the name of ClusterLogForwarder is `instance`, the collector pods are named as `instance-xxxxx`.

Reply:
thanks, updated